src/test/java/nurseybook/logic/commands/DeleteNokCommandTest.java

```diff
@@ -1,38 +1,37 @@
-package seedu.address.logic.commands;
+package nurseybook.logic.commands;
 
+import static nurseybook.logic.commands.CommandTestUtil.assertCommandFailure;
+import static nurseybook.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static nurseybook.logic.commands.CommandTestUtil.showElderlyAtIndex;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
-import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.address.logic.commands.CommandTestUtil.showElderlyAtIndex;
-import static seedu.address.testutil.TypicalElderlies.getTypicalAddressBook;
```

Comment:
Missing imports in this file as well that would be better to include at the start!